src/shared/components/query/StudySelectorStats.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import {QueryStore} from "shared/components/query/QueryStore";
+import {Observer, observer} from "mobx-react";
+import {expr} from "mobx";
+import LoadingIndicator from "shared/components/loadingIndicator/LoadingIndicator";
+
+export const StudySelectorStats: React.FunctionComponent<{ store:QueryStore }> = observer((props) => {
+    if (!props.store.cancerTypes.isPending && !props.store.cancerStudies.isPending && !props.store.profiledSamplesCount.isPending) {
+        return (<Observer>
+            {() => {
+                let numSelectedStudies = expr(() => props.store.selectableSelectedStudyIds.length);
+                return (
+                    <>
+                        <a
+                            onClick={() => {
+                                if (numSelectedStudies)
+                                    props.store.showSelectedStudiesOnly = !props.store.showSelectedStudiesOnly;
```

Comment:
a bit hard on the eye, could you put it in brackets even though its just one line?